glacierscript.py

```diff
@@ -262,6 +262,17 @@ def ensure_bitcoind_running():
 
     raise Exception("Timeout while starting bitcoin server")
 
+def require_minimum_bitcoind_version(min_version):
+    """
+    Fail if the bitcoind version in use is older than required
+    <min_version> - required minimum version in format of getnetworkinfo, i.e. 150100 for v0.15.1
+    """
+    networkinfo_str = subprocess.check_output(bitcoin_cli + "getnetworkinfo", shell=True)
+    networkinfo = json.loads(networkinfo_str)
+
+    if int(networkinfo["version"]) < min_version:
+        print "ERROR: Your bitcoind version is too old. Exiting..."
```

Comment:
can we also print the user's version and the required min_version as part of this error?

Reply:
I considered that when I wrote this, but decided against adding that complexity for several reasons:
* This check is really only for us developers. Any end-user who gets this version of GlacierScript will also get the latest version of Bitcoin Core at the same time. The chance of an end-user ever seeing this error is practically nil, unless they are taking shortcuts and not following the full protocol.
* The version number used by `getnetworkinfo` is not exactly user-friendly (e.g. 160000 for v0.16), so it would require some non-trivial code to convert to a printable form.
* Even if the end user did see this message, the only solution is to redo their App USBs to get the latest Bitcoin Core, and the process for that is the same regardless of how out-of-date their old version is.

Reply:
@bitcoinhodler: I think it'd be nice to include in case the PDF drifts from the glacierscript itself (the current version refers to v0.13, for example), or for whatever reason there is a failure with retrieving the bitcoind apt repo.That said, it's unfortunate that the version number isn't very helpful here so I think the raw output string is fine rather than writing code to format.

Reply:
Done